Resources/Prototypes/Loadouts/loadout_groups.yml

```diff
@@ -1076,12 +1076,16 @@
   loadouts:
   - HeadofSecurityJumpsuit
   - HeadofSecurityJumpskirt
-  - HeadofSecurityTurtleneck
-  - HeadofSecurityTurtleneckSkirt
+  - HeadofSecurityJumpsuitGrey # DeltaV
+  - HeadofSecurityJumpskirtGrey # DeltaV
+  - HeadofSecurityJumpsuitBlue # DeltaV
+  - HeadofSecurityJumpskirtBlue # DeltaV
+#  - HeadofSecurityTurtleneck
+#  - HeadofSecurityTurtleneckSkirt
   - HeadofSecurityFormalSuit
   - HeadofSecurityFormalSkirt
-  - HeadofSecurityParadeSuit
-  - HeadofSecurityParadeSkirt
+#  - HeadofSecurityParadeSuit # DeltaV - removed for incongruence
+#  - HeadofSecurityParadeSkirt # DeltaV - removed for incongruence
```

Comment:
I dont see why these need to be removed from the loadout, perhaps a resprite instead of removal to clean it up?

Reply:
Discussion within direction indicates we'll be moving away from the militaristic vibe. Formal uniforms are staying *for now* but may also be removed for a more consistent aesthetic.

Reply:
Me when LO parade uniform

Reply:
> Me when LO parade uniform

Also the police have parade uniforms, not just militaries

Reply:
NT ain't police. Regardless, formal uniforms are remaining in-game as previously stated - until further direction discussion

Reply:
Think private security. And the "weird mishmash megacorp" image is one we'll be refining and cutting down on in future.